src/app/portlets/dot-edit-content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.ts

```diff
@@ -104,6 +105,23 @@ export class DotEditPageToolbarComponent implements OnInit {
         return result;
     }
 
+    /**
+     * Handle the click to the locker switch
+     *
+     * @param {any} $event
+     * @memberof DotEditPageToolbarComponent
+     */
+    lockerHandler($event): void {
+        const blinkClass = 'edit-page-toolbar__cant-lock-message--blink';
+
+        if (this.locker.disabled) {
+            this.lockedPageMessage.nativeElement.classList.add(blinkClass);
+            setTimeout(() => {
+                this.lockedPageMessage.nativeElement.classList.remove(blinkClass);
+            }, 500);
```

Comment:
why dont you use the global message here?

Reply:
Be cause this is a different message, we want it to be red and we want it on the right of the title in the edit content toolbar, is not like a show/hide message it's there all the time when the page is locked.